### DISCUSSION TITLE
How to use Supabase to build a system with a marketing website, an admin portal, and a mobile app?

### DISCUSSION THREAD

**Discussion by jolleekin** [supabase/supabase#22174] [Questions]
How can I use Supabase to build a system that consists of
- A marketing website with blogs
- An admin portal for managing blogs, contents, users, and customers
- A mobile app for customers to consume the contents

The admin portal has its own set of users, which is different from the set of customers of the mobile app.

Thanks.

**Comment by cjacky475** [comment]
Hi, have you visited [https://supabase.com/](https://supabase.com/ )? It shows what it provides:

1. Database - you can store blogs, users, customers, whatever. 
2. Authentication - you can authenticate your users, admin via the Auth system.
3. Storage - you can store blog images, user profile pics, whatever. Basically files, videos or images.

Good luck.

**Reply by jolleekin** [reply]
I've been exploring Supabase recently.

My concern is how to manage authentication for users and customers separately. It looks like I need to create two Supabase projects to utilize its Auth and custom API services for cross-project storage and DB access. Is it the right approach?

Without Supabase, the system may look like this
![image](https://github.com/supabase/supabase/assets/5405376/21b8fc06-5dab-4ef0-8b6c-654e3b58f090)

How can Supabase help simplify the backend?
